src/download_db.py

```diff
@@ -1,12 +1,27 @@
-import sys
-import os
 import urllib.request
 from pathlib import Path
+from enum import Enum
+from typing import Dict
+from urllib.error import URLError
+
+
+class Database(Enum):
+    ModelSeed = "modelSeed.tsv"
+    BiGG = "BiGG.tsv"
+    VMH = "vmh.json"
+
+
+database_links: Dict[Database, str] = {
+    Database.ModelSeed: "https://raw.githubusercontent.com/ModelSEED/ModelSEEDDatabase/master/Biochemistry/compounds.tsv",
+    Database.BiGG: "http://bigg.ucsd.edu/static/namespace/bigg_models_metabolites.txt",
+    Database.VMH: "https://www.vmh.life/_api/metabolites/?format=jsonp&page_size=9999999&_dc=1670601370202&page=1&start=0&limit=9999999&callback=Ext.data.JsonP.callback19"
+}
+
 
 def create_folder(path: Path) -> Path:
```

Comment:
(2) The function only returns "." is the path creation failed. This is how its supposed to work

Reply:
(3) At the moment there is no need to create nested directory hierarchies because we want to create the Database folder always in the MeMoMe folder and relatively this will always be at the same path-location regarding the download_db.py file

Reply:
Regarding:  It will always create the folder "Databases" - not sure whether we want that, but ok.

What do you mean by this? It should only create the folder when you run it with the --download flag

Reply:
OK - I leave the design as is for now - its early stage, no need to be perfect. But in the long run, we should create the option to download the databases in a custom directory to provide the opportunity to have the databases in one place for different projects (or other reasons).